ddtrace/propagation/utils.py

```diff
@@ -1,31 +0,0 @@
-from typing import Optional
```

Comment:
do we need to keep these methods here with a deprecation warning?

Reply:
This PR is meant for v1.0, but I'll make a separate PR for deprecating first 😄

Reply:
This PR is meant for v1.0, but I'll make a separate PR for deprecating first 😄